lua/tests/helpers.lua

```diff
@@ -31,6 +31,17 @@ function helpers.setup(config)
   require("lean").setup(vim.tbl_extend("keep", config, default_config))
 end
 
+local function has_all(_, arguments)
+  local text = arguments[1]
+  local expected = arguments[2]
+  for _, string in pairs(expected) do
+    assert.has_match(string, text, nil, true)
```

Comment:
Did a nested assertion rather than returning `false` so that we could get `luassert`'s nicely formatted output on failure.